org.testeditor.web.backend.persistence/src/main/java/org/testeditor/web/backend/testexecution/TestMonitorProvider.xtend

```diff
@@ -0,0 +1,45 @@
+package org.testeditor.web.backend.testexecution
+
+import java.util.Map
+import javax.inject.Inject
+import javax.inject.Named
+
+import static org.testeditor.web.backend.testexecution.TestStatus.*
+
+class TestMonitorProvider {
```

Comment:
TestMonitorProvider suggests that it provides a TestMonitor (through a get). How about just TestStatusMapper or something similar. You could also provide a short class doc.